sdk/core/abort-controller/package.json

```diff
@@ -3,43 +3,46 @@
   "sdk-type": "client",
   "version": "2.0.1",
   "description": "Microsoft Azure SDK for JavaScript - Aborter",
-  "main": "./dist/index.js",
-  "module": "dist-esm/src/index.js",
-  "scripts": {
-    "build:samples": "echo Obsolete",
-    "build:test": "tsc -p . && dev-tool run bundle",
-    "build": "npm run clean && tsc -p . && dev-tool run bundle && api-extractor run --local",
-    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist dist-* temp types *.tgz *.log",
-    "execute:samples": "echo skipped",
-    "extract-api": "tsc -p . && api-extractor run --local",
-    "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
-    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json api-extractor.json src test --ext .ts",
-    "pack": "npm pack 2>&1",
-    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run clean && tsc -p . && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run clean && tsc -p . && npm run unit-test:node && dev-tool run bundle && npm run unit-test:browser && npm run integration-test",
-    "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "dev-tool run test:node-ts-input --no-test-proxy=true",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/abort-controller/README.md",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "sideEffects": false,
+  "type": "module",
+  "main": "./dist/commonjs/index.js",
```

Comment:
Older environments may pick up on the main, however newer environments should only look at the top level exports instead since that supersedes the main/types